pkg/deviceplugin/manager.go

```diff
@@ -92,13 +101,28 @@ func NewNicManager(
 
 // Initialises the set of devices to advertise to kubernetes
 func (manager *NicManager) initDevices() {
+
+	// Gets a list of all numa nodes of which there is an associated sfc nic,
+	// this isn't particularly helpful when you have nics on different numa
+	// nodes which are intended for different purposes, but this is basically a
+	// pathological case due to how we only advertise an "onload" device rather
+	// than a "real" one.
+	numaNodes := []*pluginapi.NUMANode{}
```

Comment:
Do we need to ensure uniqueness in the `numaNodes` slice? Either way, it would be nice to capture it as a unit test...